src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceGenerator.cs

```diff
@@ -11,7 +11,7 @@
 using System.Collections.Immutable;
 using System.IO;
 using System.Linq;
-using System.Runtime.CompilerServices;
+using System.Net.Mime;
```

Comment:
Is this using needed?

Reply:
Ah, nope. Not entirely sure how that got added...